lib/main.dart

```diff
@@ -1,16 +1,12 @@
 import 'package:flutter/material.dart';
+import 'package:pantry/scan_screen.dart';
 import 'home_screen.dart';
 import 'login_screen.dart';
-import 'package:intl/intl.dart';
-import 'package:datetime_picker_formfield/datetime_picker_formfield.dart';
 
 void main() => runApp(new MyApp());
 
-String getDate() {
-  var today = new DateTime.now();
-  var formatter = new DateFormat('MM/dd/yyyy');
-  String formattedDate = formatter.format(today);
-  return formattedDate;
+class GlobalData {
+  static String auth;
```

Comment:
Took out incorrect global date formatting and put a global auth token string available for all screens